src/app/pages/contact/contact.component.html

```diff
@@ -1,31 +1,34 @@
-<h1>Contact: {{ contact.name }}</h1>
-
 <div class="contact-container">
-  <div *ngIf="contact.address">Address: {{ contact.address }}</div>
+  <h1>{{ contact.name }}</h1>
+  <div class="info" *ngIf="contact.address">Address: {{ contact.address }}</div>
 
-  <div *ngIf="contact.mobile || contact.work || contact.home">
+  <div class="info" *ngIf="contact.mobile || contact.work || contact.home">
     Phone:
-    <div *ngIf="contact.mobile">Mobile: {{ contact.mobile }}</div>
+    <div class="phone-info" *ngIf="contact.mobile">
```

Comment:
Try to keep the formatting of the HTML consistent in this file as well. I tend to prefer not one line-ing the html here since it's more readable that way.